packages/core/scripts/severless-cloud-run/test/CloudRunHub.js

```diff
@@ -0,0 +1,344 @@
+          TOKEN_PRICE_FEED_CONFIG: defaultUniswapPricefeedConfig
+        }
+      }
+    };
+    // Set env variables for the hub to pull from. Add the startingBlockNumber and the hubConfig.
+    setEnvironmentVariable(`lastQueriedBlockNumber-${testConfigFile}`, startingBlockNumber);
+    setEnvironmentVariable(`${testBucket}-${testConfigFile}`, JSON.stringify(hubConfig));
+
+    const validBody = { bucket: testBucket, configFile: testConfigFile };
+
+    const validResponse = await sendHubRequest(validBody);
+    assert.equal(validResponse.res.statusCode, 200); // no error code
+    assert.isTrue(validResponse.res.text.includes("All calls returned correctly")); // Final text in monitor loop.
+    assert.isTrue(lastSpyLogIncludes(hubSpy, "All calls returned correctly")); // The hub should have exited correctly.
+    assert.isTrue(spyLogIncludes(hubSpy, -3, `"botsExecuted":${JSON.stringify(Object.keys(hubConfig))}`)); // all bots within the config should have been reported to be executed.
+    assert.isTrue(lastSpyLogIncludes(spokeSpy, "Process exited with no error")); // The spoke should have exited correctly.
+    assert.isTrue(lastSpyLogIncludes(spokeSpy, `startingBlock: ${startingBlockNumber + 1}`)); // The spoke should have the correct starting block number.
+  });
+  it("Cloud Run Hub can correctly execute multiple bots in parallel", async function() {
+    // Set up the environment for testing. For these tests the hub is tested in `localStorage` mode where it will
+    // read in hub configs and previous block numbers from the local storage of machine. This execution mode would be
+    // used by a user running the hub-spoke on their local machine.
+    const testBucket = "test-bucket"; // name of the config bucket.
+    const testConfigFile = "test-config-file"; // name of the config file.
+    const startingBlockNumber = await web3.eth.getBlockNumber(); // block number to search from for monitor
+
+    const hubConfig = {
+      testServerlessMonitor: {
+        cloudRunCommand: `node ${path.resolve(__dirname)}/../../../../monitors/index.js --network test`,
+        environmentVariables: {
+          BOT_IDENTIFIER: "test-serverless-monitor",
+          CUSTOM_NODE_URL: web3.currentProvider.host,
+          POLLING_DELAY: 0,
+          EMP_ADDRESS: emp.address,
+          TOKEN_PRICE_FEED_CONFIG: defaultUniswapPricefeedConfig
+        }
+      },
+      testCloudRunLiquidator: {
+        cloudRunCommand: `node ${path.resolve(__dirname)}/../../../../liquidator/index.js --network test`,
+        environmentVariables: {
+          BOT_IDENTIFIER: "test-serverless-liquidator",
+          CUSTOM_NODE_URL: web3.currentProvider.host,
+          POLLING_DELAY: 0,
+          EMP_ADDRESS: emp.address,
+          PRICE_FEED_CONFIG: defaultUniswapPricefeedConfig
+        }
+      },
+      testCloudRunDisputer: {
+        cloudRunCommand: `node ${path.resolve(__dirname)}/../../../../disputer/index.js --network test`,
```

Comment:
nit: could you do something like this instead:
```js
cloudRunCommand: `yarn disputer --network test`
```
?

I like this a little better because it doesn't depend on the dir structure. My understanding is that yarn automatically resolves all of your workspace `bin` files (which we have for the bots and the CLI).

I think this would also work for the others above, too.

Reply:
ye for sure! that's awesome. I was not sure that this functionality was done 100% but that's great that it works now. We could consider updating our docs to use this syntax as well. This is really the power of having the mono repo with `index.js` entry points in all our packages